modules/esm/components/ProgressRing.tsx

```diff
@@ -50,13 +48,7 @@ const ProgressRing = ({
           cy={radius}
         ></circle>
         <text x="50%" y="48%" textAnchor="middle" fill="#434358" fontSize="18px" dy=".3em">
-          {totalStaked ? (
-            `${formatValue(totalStaked, 'wad', 6)}      `
-          ) : (
-            <Box pl="14px" pr="14px">
-              <div ref={loader} />
-            </Box>
-          )}
+          {totalStaked ? `${formatValue(totalStaked, 'wad', 6)}      ` : '---      '}
```

Comment:
is this supposed to have that many spaces after the ---?

Reply:
Seems like the format is still correct without the extra spaces, I'll remove those